testing-junit4/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+apply from: '../gradle/publication.gradle'
+
+android {
+    namespace 'com.bumble.appyx.testing'
```

Comment:
This could cause a collision potentially

Reply:
Please keep it consistent with the manifest 'com.bumble.appyx.testing.junit4'

Reply:
Sorry, missed this, thanks!